### PR TITLE
Bugfixes/improvements

### PR DESCRIPTION
**Pull request by fryette**

### CODE REVIEW

File: ecommers/lib/ui/widgets/product_item/product_item_small.dart [hunk 1]
```diff
@@ -35,10 +35,7 @@ class ProductItemSmall extends ProductItemBase {
       children: <Widget>[
         Expanded(
           flex: 3,
-          child: HeroImage(
-            tag: id,
-            imagePath: assetImagePath,
-          ),
+          child: CachedImage(imagePath: assetImagePath),
```
Comment (fryette) [inline ecommers/lib/ui/widgets/product_item/product_item_small.dart:38]: Here we don't need hero image

File: ecommers/lib/core/services/navigation/navigation_service.dart [hunk 2]
```diff
@@ -69,6 +69,6 @@ class NavigationService {
   }
 
   Route<dynamic> _getRoute(Widget widget) {
-    return CupertinoPageRoute(builder: (_) => widget);
+    return MaterialPageRoute(builder: (_) => widget);
```
Comment (Platton96) [inline ecommers/lib/core/services/navigation/navigation_service.dart]: why did you change  PageRoute?

Comment (fryette) [reply ecommers/lib/core/services/navigation/navigation_service.dart]: returned

File: ecommers/lib/ui/pages/note_page.dart [hunk 3]
```diff
@@ -32,22 +30,24 @@ class NotePage extends StatelessWidget {
...
                   style: titleStyle,
                 ),
               ),
               centerTitle: true,
-              background: CachedImage(
-                imagePath: model.imageUrl,
-                boxFit: BoxFit.cover,
-              ),
+              background: Hero(
+                  tag: model.imageUrl,
```
Comment (Platton96) [inline ecommers/lib/ui/pages/note_page.dart]: use custom widget  HeroImage

File: ecommers/lib/ui/widgets/product_item/product_item_small.dart [hunk 4]
```diff
@@ -35,10 +35,7 @@ class ProductItemSmall extends ProductItemBase {
       children: <Widget>[
         Expanded(
           flex: 3,
-          child: HeroImage(
-            tag: id,
-            imagePath: assetImagePath,
```
Comment (Platton96) [inline ecommers/lib/ui/widgets/product_item/product_item_small.dart:40]: why did you decide to remove heroImage

Comment (fryette) [reply ecommers/lib/ui/widgets/product_item/product_item_small.dart:40]: discussed

**Review by Platton96 — APPROVED** [review]
